src/components/OcSelect.vue

```diff
@@ -147,16 +166,18 @@ export default {
 ```
 
 ### Using slots to display complex options
-Sometimes we need to display more complex options. This can include e.g. an option with a title and a description. To still display all those values exactly as we want to, we need to use scoped slot called `option`.
-We can then retrieve all the values that we want to display from the slots parametres. It is important to specify the `label` prop on the `oc-select` component which will specify which key should be used as the option label.
+Sometimes we need to display more complex options. This can include e.g. an option with a title and a description. 
+To still display all those values exactly as we want to, we need to use scoped slot called `option`.
+We can then retrieve all the values that we want to display from the slots parametres. 
+It is important to use a `label` key inside the passed object since `oc-select` will look for it.
```

Comment:
This sounds rather complicated. Why don't we introduce a `option-label` property that will be passed as `label` to vue-select?

Reply:
Sounds like a great idea actually, let me take a look 🕵🏽

Reply:
Or get it from `$attrs` fwiw